test/functional/fixtures/lua/test_plug/submodule/health.lua

```diff
@@ -1,5 +1,5 @@
 local M = {}
-local health = require("health")
+local health = require("health")  -- change to use vim.health on 0.9
```

Comment:
? why not change it in this PR?

Reply:
Kinda to show it still worked the old way 😁 will remove!

Reply:
Done 👍